netif.pyx

```diff
@@ -536,10 +536,14 @@ class InterfaceAddress(object):
     def __hash__(self):
         return hash((self.af, self.address, self.netmask, self.broadcast, self.dest_address))
 
-    def __getstate__(self):
+    def __getstate__(self, stats=False):
         ret = {
             'type': self.af.name,
-            'address': self.address.address if type(self.address) is LinkAddress else str(self.address),
+            'address': self.address.address if type(self.address) is LinkAddress else str(self.address)
+        }
+
+        if stats:
+            ret['stats'] = {
```

Comment:
bad indentation here? PEP8

Reply:
I mean, below his line

Reply:
ouch, thank you for spotting. Please have a look again.